### DISCUSSION TITLE
Command to open "languages.toml" similar to "config-open"

### DISCUSSION THREAD

**Discussion by IvanGoncharov** [helix-editor/helix#10401] [General]
I'm messing a lot with `languages.toml` after switching to helix as my primary editor.
It would be great to have some way to open/reload it quickly.

Maybe just add it as an argument to config-open, e.g., `config-open languages`
If, in the future other configuration files (e.g. `plugins.toml`) it would be useful to have one command with different arguments.

A more radical change would be to make "config-open" open a file picker in the config directory.

**Comment by the-mikedavis** [comment]
This was the idea of https://github.com/helix-editor/helix/issues/3206 when opened. A command for opening language config may be more confusing than helpful since there are three possible language configurations and not all of them are editable.

**Comment by gerep** [comment]
You can create a shortcut in your `config.toml`.

```toml
[keys.normal]
"C-l" = ":open ~/.config/helix/languages.toml"
```

And pressing `ctrl+l` will open the file as a new buffer.

**Comment by danriedl** [comment]
+1, also having the same thing within the "config-open-workspace" to open the language file from there.
I have to quit helix and open it directly because of course `.helix` is in `.gitignore`.
